sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/CryptographyClient.cs

```diff
@@ -139,7 +139,7 @@ internal CryptographyClient(Uri keyId, KeyVaultPipeline pipeline)
         public virtual string KeyId => _keyId.ToString();
 
         /// <summary>
-        /// Encrypts the specified plain text.
+        /// Encrypts the specified plaintext.
```

Comment:
nit: We're inconsistent between this and "Encrypts plaintext". I actually like this one a bit better.

Reply:
That was intentional. You're not directly specifying the plaintext in the overload.